src/main/kotlin/org/jitsi/nlj/util/PacketInfoQueue.kt

```diff
@@ -29,23 +29,8 @@ class PacketInfoQueue(
     executor: ExecutorService,
     handler: (PacketInfo) -> Boolean,
     capacity: Int = 1024
-) : PacketQueue<PacketInfo>(capacity, false, null, id, handler, executor) {
+) : PacketQueue<PacketInfo>(capacity, null, id, handler, executor) {
```

Comment:
While in here changing things, it'd be nice to have named args here

Reply:
Unfortunately this is a Java method it's calling, so named parameters aren't supported.

Reply:
Oh, didn't realize that was dependent on the callee. Bummer.